plugins/aver-hid/fu-aver-hid-device.c

```diff
@@ -59,7 +59,7 @@ fu_aver_hid_device_transfer(FuAverHidDevice *self, GByteArray *req, GByteArray *
 }
 
 static gboolean
-fu_aver_hid_device_poll(FuDevice *device, GError **error)
+fu_aver_hid_device_ensure_status(FuDevice *device, GError **error)
```

Comment:
if this is an internal function (rather than a callback) it should use `FuAverHidDevice *self` as a first parameter, rather than `FuDevice *device` -- this means you need to add `FU_AVER_HID_DEVICE(device)` to `fu_aver_hid_device_setup` as well.